libs/components/config/src/lib/params.ts

```diff
@@ -178,44 +178,103 @@ export class SkyAppRuntimeConfigParams {
 
   /**
    * Adds the current params to the supplied link URL.
+   * @param url The url to update.
+   * @param queryParams Optional query parameters to include in the constructed url.
    */
-  public getLinkUrl(url: string): string {
+  public getLinkUrl(
+    url: string,
+    queryParams?: Record<string, string | number | boolean | null | undefined>
```

Comment:
Should this use the `SkyHrefQueryParams` interface that was added in this PR?

Reply:
That file was created in `@skyux/router` and `@skyux/config` does not have a peer dependency on router. I could put an unexported type at the top of the file or create a new public type, but I was trying to avoid adding too much to the API.